sklearn/base.py

```diff
@@ -354,7 +354,9 @@ def score(self, X, y, sample_weight=None):
         Parameters
         ----------
         X : array-like, shape = (n_samples, n_features)
-            Test samples.
+            Test samples. For some estimators this may be a
+            precomputed kernel matrix instead, shape = (n_samples,
+            n_samples).
```

Comment:
I think this should be n_samples_fitted, like in predict.

Reply:
Fixed. Thanks.